src/minicraft/entity/mob/Creeper.java

```diff
@@ -92,33 +93,33 @@ public void tick() {
 						Mob mob = (Mob) entity;
 						int distx = Math.abs(mob.x - x);
 						int disty = Math.abs(mob.y - y);
-						float distDiag = (float) Math.sqrt(distx * distx + disty * disty);
+						float distDiag = (float) Math.sqrt(distx^2 + disty^2);

```

Comment:
The `^` operator is a bitwise XOR. Undo this change.

Reply:
1: I normally change them when I notice inconsistency. Don't want to change all of it at once, since they lead to messy commits.
2: See last commit; yd and xd are deltas, and xmov and ymov are distances. These two are different in some functions.
3: Fair enough

Reply:
> 
> 
> The `^` operator is a bitwise XOR. Undo this change.

Oh, my bad